syne_tune/backend/sagemaker_backend/sagemaker_backend.py

```diff
@@ -132,11 +132,10 @@ def np_encoder(object):
         return json.loads(json.dumps(dict, default=np_encoder))
 
     def _checkpoint_s3_uri_for_trial(self, trial_id: int) -> str:
-        res_path = Path(self.s3_path)
+        res_path = self.s3_path
```

Comment:
If the last character is '/', you'd have to remove it.

An alternative is to strip off the "s3://" prefix and then use Path

Reply:
the `self.s3_path.rstrip('/')`is already in the init of the class.